### ISSUE TITLE
Deduplicate the Datatype Property declarations

### ISSUE THREAD

**Issue by rhohimer** [oasis-open/tac-ontology#37]
When the stix.owl ontology was divided into many namespaces there were multiple definitions asserted for the properties in each namespace. For instance, multiple declarations for "name" or "value".  Basically, common properties were declared in each namespace.

**Comment by mateusdz** [comment]
After the ontology refactors, we have reduced the namespaces count to one (stix namespace). All common properties are collected in one file and reused in all STIX objects. However, there exist several properties (that are not common properties, but) many objects are using, e.g., "name" and "description."

**Comment by mateusdz** [comment]
In file.owl, the file object is defined together with its extensions. Here both the base class and the extensions have the same property name and values in two properties, "name" and "size." In this case, to avoid conflicts, I have defined these properties only once without the descriptions of the properties.

**Comment by rhohimer** [comment]
This is a duplicate of Issue #20 regarding deduplication of datatype properties.
